app/models/job_type.rb

```diff
@@ -10,7 +10,7 @@ class JobType < ApplicationRecord
   private
 
   def nullify_blanks
-    environment_variables = "{}" if environment_variables&.empty?
-    working_directory = nil if working_directory&.empty?
+    self.environment_variables = "{}" if environment_variables&.empty?
+    self.working_directory = nil if working_directory&.empty?
```

Comment:
can we use the format `update(working_directory: nil)` to make sure the updates save? From what I learned yesterday, we probably aren't actually committing the changes to the db if we don't since we never call `save`. and also that way it's clearer it's an activerecord property.